README.md

```diff
@@ -2,48 +2,49 @@
-| [listmempoolfunds][listmempoolfunds] | Track unconfirmed wallet deposits                                                           |
-| [monitor][monitor]                   | helps you analyze the health of your peers and channels                                     |
-| [nloop][nloop]                       | Generic Lightning Loop for boltz                                                            |
-| [paythrough][paythrough]             | Pay an invoice through a specific channel, regardless of better routes                      |
-| [persistent-channels][pers-chans]    | Maintains a number of channels to peers                                                     |
-| [poncho][poncho]                     | Turns CLN into a [hosted channels][blip12] provider                                         |
-| [pruning][pruning]                   | This plugin manages pruning of bitcoind such that it can always sync                        |
-| [python-teos][python-teos]           | The Eye of Satoshi - Lightning Watchtower                                                   |
-| [rebalance][rebalance]               | Keeps your channels balanced                                                                |
-| [reckless][reckless]                 | An **experimental** plugin manager (search/install plugins)                                 |
-| [sauron][sauron]                     | A Bitcoin backend relying on [Esplora][esplora]'s API                                       |
-| [sitzprobe][sitzprobe]               | A Lightning Network payment rehearsal utility                                               |
-| [sling][sling]                       | Rebalance your channels with smart rules and built-in background tasks                      |
-| [sparko][sparko]                     | RPC over HTTP with fine-grained permissions, SSE and spark-wallet support                   |
-| [summars][summars]                   | Print configurable summary of node, channels and optionally forwards, invoices, payments    |
-| [trustedcoin][trustedcoin]           | Replace your Bitcoin Core with data from public block explorers                             |
-| [watchtower][watchtower-client]      | Watchtower client for The Eye of Satoshi                                                    |
-| [webhook][webhook]                   | Dispatches webhooks based from [event notifications][event-notifications]                   |
-| [zmq][zmq]                           | Publishes notifications via [ZeroMQ][zmq-home] to configured endpoints                      |
+| Name                                 | Short description                                                                           | Main ![GitHub Release](https://img.shields.io/github/v/release/ElementsProject/lightning?label=%20&color=393D47) | Nigthly ![Static Badge](https://img.shields.io/badge/master-master?color=393D47) |
```

Comment:
```suggestion
| Name                                 | Short description                                                                           | Main ![GitHub Release](https://img.shields.io/github/v/release/ElementsProject/lightning?label=%20&color=393D47) | Nightly ![Static Badge](https://img.shields.io/badge/master-master?color=393D47) |
```

Reply:
I would change both Main and Nightly to CLN. So it would read "CLN v24.02.1" and "CLN master" right now. I think this makes it way more clear.

Reply:
Done.